crates/bevy_asset/src/path.rs

```diff
@@ -1026,4 +1070,30 @@ mod tests {
         let result = AssetPath::from("asset.Custom");
         assert_eq!(result.get_full_extension(), Some("Custom".to_string()));
     }
+
+    #[test]
+    fn test_trailing_slash_equality() {
+        assert_eq!(AssetPath::from("a/b/"), AssetPath::from("a/b"));
```

Comment:
Does an empty path `""` test as equal to `"/"`?

Reply:
I don't think that should be the case. Added a test for that just in case